### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by indcoder**

### CODE REVIEW

**Comment by indcoder** [comment]
@googlebot I signed it!

File: css/main.css [hunk 1]
```diff
@@ -395,16 +395,839 @@ SOFTWARE.
...
+html {
```
Comment (cramforce) [inline css/main.css:398]: Could you revert this. I'd like to keep the copy-pasted CSS unchanged.

**Review by cramforce — COMMENTED** [review]
Thanks!

File: css/main.css (hunk 1 above)
Comment (indcoder) [reply css/main.css]: Sorry about that. Prettier is enable on file save, hence the changes across.

**Review by cramforce — APPROVED** [review]
Thanks!

**Pull request merged**